src/js/models/guide.js

```diff
@@ -39,6 +39,13 @@ define(function (require, exports, module) {
         documentID: null,
 
         /**
+         * True if the guide belongs to the document, false if it's an artboard guide
+         *
+         * @type {boolean}
+         */
+        isGlobal: null,
```

Comment:
Is there a better name for this? Global is probably not the right word here. How about `isLayerGuide` or `isDocumentGuide`?